### PR TITLE
✨ [ads2.bid] amp-ad ads2bid implementation

### PR DESCRIPTION
**Pull request by github-profitclicks**
amp-ad ads2bid implementation

cc ampproject/wg-monetization // from docs

### CODE REVIEW

**Comment by github-profitclicks** [comment]
@lannka Hello! During what time can we expect a approval?

**Review by lannka — APPROVED** [review]

**Comment by github-profitclicks** [comment]
Hello! @lannka or other **owners**, what is that mean in CircleCI? That error on first iteration.
```
#!/bin/bash -eo pipefail
curl -sS https://raw.githubusercontent.com/ampproject/amphtml/main/.circleci/compute_merge_commit.sh | bash

bash: line 1: 404:: command not found

Exited with code exit status 127

CircleCI received exit code 127
```

**Comment by github-profitclicks** [comment]
@calebcordry Hello! I see you in other MR's. Can you help us?

File: ads/vendors/ads2bid.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!---
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/vendors/ads2bid.md:2]: nit: 2021

File: ads/vendors/ads2bid.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/vendors/ads2bid.js:2]: nit: 2021

**Comment by calebcordry** [comment]
Sorry for the delay here. Since it has been a while you may need to sync to the latest main branch. Can you try that please?

File: ads/vendors/ads2bid.md (hunk 1 above)
Comment (github-profitclicks) [reply ads/vendors/ads2bid.md]: done

File: ads/vendors/ads2bid.js (hunk 2 above)
Comment (github-profitclicks) [reply ads/vendors/ads2bid.js]: done

**Comment by github-profitclicks** [comment]
@calebcordry, thank you so much for the answer! 
The branch has the latest main version, but I synced again (maybe I don't quite understand, the main branch is set to upstream/main and just I do a git pull to our. That is ok?). As far as I could see, unlike recent pull requests - our flow CircleCI is very different. Maybe that's the problem?

**Comment by github-profitclicks** [comment]
@calebcordry, could you give me an answer, please? What is the problem with CircleCI?

```
#!/bin/bash -eo pipefail
curl -sS https://raw.githubusercontent.com/ampproject/amphtml/main/.circleci/compute_merge_commit.sh | bash
bash: line 1: 404:: command not found

Exited with code exit status 127
CircleCI received exit code 127
```

**Comment by github-profitclicks** [comment]
Maybe we should just make a new merge request? This has been going on for a very long time...

**Comment by calebcordry** [comment]
Sorry missed this. Something like:
```sh
git checkout main
git pull upstream main
git checkout <your_branch>
git rebase main
git push origin <your_branch>
```
should sync everything to HEAD of our main branch. This fixes most CI errors.

**Comment by github-profitclicks** [comment]
@calebcordry thank you! But now we are facing problems. Could you explain the reason?

**1. End-to-end test**
```
amp-bind chrome Standalone environment "before each" hook for "should support binding to src"

"before each" hook for "should support binding to src"
/home/circleci/project/test/e2e/test-amp-bind-iframe.js
Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/circleci/project/test/e2e/test-amp-bind-iframe.js)
    at listOnTimeout (node:internal/timers:557:17)
    at processTimers (node:internal/timers:500:7)
```

2. codecov/patch **codec failure/correction after 1 second — 72.97% of differences (target 75.00%)**

3. Owners-Check **Recommended Reviewers: Ridgewell**

**Comment by calebcordry** [comment]
Yeah it looks like that didnt quite work as expected. It might be easier to start a new one now, or I can try to walk you through cleaning up the commits?

**Comment by github-profitclicks** [comment]
@calebcordry thanks. We created new MR. Link: https://github.com/ampproject/amphtml/pull/37501

**Pull request closed without merging**